addons/website_sale/models/product.py

```diff
@@ -13,7 +13,10 @@ class ProductRibbon(models.Model):
     _name = "product.ribbon"
     _description = 'Product ribbon'
 
-    html = fields.Char(string='Ribbon html', required=True)
+    def name_get(self):
+        return [(ribbon.id, '%s (#%d)' % (tools.html2plaintext(ribbon.html), ribbon.id)) for ribbon in self]
```

Comment:
Maybe 
```suggestion
        return [(ribbon.id, '%s (#%d)' % (_(tools.html2plaintext(ribbon.html)), ribbon.id)) for ribbon in self]
```
Since It is translatable now

Reply:
Not sure how it will works...

Terms are not exported since dynamic terms `ribbon.html` and `html2plaintext` could alter the text.
So it is probably overkill to try to handle the translation here. '... (#1)' will make it universally understandable ;)